data/modelCuration/Rxn_unique_subsystem.tsv

```diff
@@ -0,0 +1,4059 @@
+r_2656	MLCL (2-16:1, 3-16:0, 4-18:1):PC (1-16:0, 2-18:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:0, 4-18:1)[mm] + phosphatidylcholine (1-16:0, 2-18:1)[mm] <=> cardiolipin (1-18:1, 2-16:1, 3-16:0, 4-18:1)[mm] + 1-acylglycerophosphocholine (16:0)[mm]	YPR140W	glycerophospholipid metabolism
+r_2657	MLCL (2-16:1, 3-16:0, 4-18:1):PC (1-16:1, 2-18:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:0, 4-18:1)[mm] + phosphatidylcholine (1-16:1, 2-18:1)[mm] <=> cardiolipin (1-18:1, 2-16:1, 3-16:0, 4-18:1)[mm] + 1-acylglycerophosphocholine (16:1)[mm]	YPR140W	glycerophospholipid metabolism
+r_2658	MLCL (2-16:1, 3-16:0, 4-18:1):PC (1-18:0, 2-18:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:0, 4-18:1)[mm] + phosphatidylcholine (1-18:0, 2-18:1)[mm] <=> cardiolipin (1-18:1, 2-16:1, 3-16:0, 4-18:1)[mm] + 1-acylglycerophosphocholine (18:0)[mm]	YPR140W	glycerophospholipid metabolism
+r_2659	MLCL (2-16:1, 3-16:0, 4-18:1):PC (1-18:1, 2-18:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:0, 4-18:1)[mm] + phosphatidylcholine (1-18:1, 2-18:1)[mm] <=> cardiolipin (1-18:1, 2-16:1, 3-16:0, 4-18:1)[mm] + 1-acylglycerophosphocholine (18:1)[mm]	YPR140W	glycerophospholipid metabolism
+r_2660	MLCL (2-16:1, 3-16:1, 4-18:1):PC (1-16:0, 2-16:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:1, 4-18:1)[mm] + phosphatidylcholine (1-16:0, 2-16:1)[mm] <=> cardiolipin (1-16:1, 2-16:1, 3-16:1, 4-18:1)[mm] + 1-acylglycerophosphocholine (16:0)[mm]	YPR140W	glycerophospholipid metabolism
+r_2661	MLCL (2-16:1, 3-16:1, 4-18:1):PC (1-16:1, 2-16:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:1, 4-18:1)[mm] + phosphatidylcholine (1-16:1, 2-16:1)[mm] <=> cardiolipin (1-16:1, 2-16:1, 3-16:1, 4-18:1)[mm] + 1-acylglycerophosphocholine (16:1)[mm]	YPR140W	glycerophospholipid metabolism
+r_2662	MLCL (2-16:1, 3-16:1, 4-18:1):PC (1-18:0, 2-16:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:1, 4-18:1)[mm] + phosphatidylcholine (1-18:0, 2-16:1)[mm] <=> cardiolipin (1-16:1, 2-16:1, 3-16:1, 4-18:1)[mm] + 1-acylglycerophosphocholine (18:0)[mm]	YPR140W	glycerophospholipid metabolism
+r_2663	MLCL (2-16:1, 3-16:1, 4-18:1):PC (1-18:1, 2-16:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:1, 4-18:1)[mm] + phosphatidylcholine (1-18:1, 2-16:1)[mm] <=> cardiolipin (1-16:1, 2-16:1, 3-16:1, 4-18:1)[mm] + 1-acylglycerophosphocholine (18:1)[mm]	YPR140W	glycerophospholipid metabolism
+r_2664	MLCL (2-16:1, 3-16:1, 4-18:1):PC (1-16:0, 2-18:1) acyltransferase, mitochondrial membrane	monolysocardiolipin (2-16:1, 3-16:1, 4-18:1)[mm] + phosphatidylcholine (1-16:0, 2-18:1)[mm] <=> cardiolipin (1-18:1, 2-16:1, 3-16:1, 4-18:1)[mm] + 1-acylglyc
```

Comment:
Here and elsewhere, please write out `FA ester pathway` as `Fatty acid ester pathway`, abbreviations are in principle not used in subSystem names.